templates/tableau-cluster-linux-master.template

```diff
@@ -0,0 +1,706 @@
+                        ]
+                    },
+                    "PrivateSubnet3ACIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PrivateSubnet3CIDR"
+                        ]
+                    },
+                    "PublicSubnet1CIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PublicSubnet1CIDR"
+                        ]
+                    },
+                    "PublicSubnet2CIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PublicSubnet2CIDR"
+                        ]
+                    },
+                    "PublicSubnet3CIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "PublicSubnet3CIDR"
+                        ]
+                    },
+                    "VPCCIDR":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "NetworkConfiguration",
+                            "VPCCIDR"
+                        ]
+                    }
+                }
+            }
+        },
+        "BastionHost":{  
+            "Type":"AWS::CloudFormation::Stack",
+            "DependsOn":[  
+                "VPCStack"
+            ],
+            "Properties":{  
+                "TemplateURL":{  
+                    "Fn::Sub": "https://${QSS3BucketName}.s3.amazonaws.com/${QSS3KeyPrefix}templates/linux-bastion.template"
+                },
+                "Parameters":{  
+                    "BastionInstanceType":{  
+                        "Fn::FindInMap":[  
+                            "DefaultConfiguration",
+                            "MachineConfiguration",
+                            "BastionInstanceType"
+                        ]
+                    },
+                    "KeyPairName":{  
+                        "Ref":"KeyPairName"
+                    },
+                    "RemoteAccessCIDR":{  
+                        "Ref":"SourceCIDR"
+                    },
+                    "PublicSubnet1ID":{  
+                        "Fn::GetAtt":[  
+                            "VPCStack",
+                            "Outputs.PublicSubnet1ID"
+                        ]
+                    },
+                    "PublicSubnet2ID":{  
+                        "Fn::GetAtt":[  
+                            "VPCStack",
+                            "Outputs.PublicSubnet2ID"
+                        ]
+                    },
+                    "VPCID":{  
```

Comment:
You'll want to include the QSS3BucketName and QSS3KeyPrefix parameters like in: 
https://github.com/aws-quickstart/quickstart-tableau-awsdata/blob/master/templates/tableau-awsdata-master.template#L958-L963